server/src/main/java/io/spine/server/enrich/Enricher.java

```diff
@@ -46,13 +46,9 @@ protected Enricher(EnricherBuilder<M, C, ?> builder) {
     /**
      * Enriches the passed message if it can be enriched. Otherwise, returns the passed instance.
      *
-     * <p>An message can be enriched if the following conditions are met:
+     * <p>An message can be enriched if the following condition is met:
      *
      * <ol>
```

Comment:
Why are we having the ordered list at all, assuming that there is only one condition left?